Framework/Crystal/src/SCDCalibratePanels.cpp

```diff
@@ -478,49 +488,48 @@ bool GoodStart(const PeaksWorkspace_sptr &peaksWs, double a, double b, double c,
                double alpha, double beta, double gamma, double tolerance) {
   // put together a list of indexed peaks
   std::vector<V3D> hkl;
-  hkl.reserve(peaksWs->getNumberPeaks());
+  int nPeaks = peaksWs->getNumberPeaks();
+  hkl.reserve(nPeaks);
   std::vector<V3D> qVecs;
-  qVecs.reserve(peaksWs->getNumberPeaks());
-  for (int i = 0; i < peaksWs->getNumberPeaks(); i++) {
+  qVecs.reserve(nPeaks);
```

Comment:
`resize()` and then assignments can be faster than `reserve()` and then push_backs. But I don't think it makes any difference in this case, only when you have large amounts of items.